design/one-pager-managed-resource-api-design.md

```diff
@@ -379,6 +379,149 @@ type SubnetworkParameters struct {
 }
 ```
 
+### Exceptions and Edge Cases
```

Comment:
I think people reading this document are unlikely to go check whether there is a section about exceptions. If we have the two sections here in their corresponding sections as an addition, they'll more likely be picked up by the readers.